# SetGain.srv
# Request part
float64 gain
---
# Response part
bool success
string message